### ISSUE TITLE
Fix admin language script editing from replacing \n with \r\n

### ISSUE THREAD

**Issue by spacetag** [unoacm/code_court#156]

**Comment by spacetag** [comment]
Maybe fix this on problem input/output, too?

**Comment by spacetag** [comment]
Maybe fix this on input/output too?

**Comment by spacetag** [comment]
So, I fixed the \r\n (CRLF) -> \n issue.

But, it didn't seem to matter in a recent version of code_court.

I checked out an older version of code_court (3ee41d6), and that's when making the change made a difference:

Without the change, when you edit the python run script online and save it, causing \r\n's to be inserted, then try to run a python program on the defendant frontend, this error occurs comes back on the run output:
`rpc error: code = 2 desc = oci runtime error: exec failed: container_linux.go:247: starting container process caused "no such file or directory"`

But, I didn't see that same issue in a recent version of code_court.
